stacks/serverless/pixel.prx.org.yml

```diff
@@ -0,0 +1,228 @@
+# stacks/serverless/pixel.prx.org.yml
+AWSTemplateFormatVersion: "2010-09-09"
+Description: API Gateway pixel tracker
+Conditions:
+  CreateProductionResources: !Equals [!Ref EnvironmentType, Production]
+Parameters:
+  OpsWarnMessagesSnsTopicArn:
+    Type: String
+  OpsErrorMessagesSnsTopicArn:
+    Type: String
+  CodeS3Bucket:
+    Type: String
+  CodeS3ObjectVersion:
+    Type: String
+  EnvironmentType:
+    Type: String
+Mappings:
+  EnvironmentTypeMap:
+    Testing:
+      DomainName: "pixel.test.prx.tech"
+      HostedZoneName: "prx.tech."
+      ValidationDomain: "prx.tech"
+    Staging:
+      DomainName: "pixel.staging.prx.tech"
+      HostedZoneName: "prx.tech."
+      ValidationDomain: "prx.tech"
+    Production:
+      DomainName: "pixel.prx.org"
+      HostedZoneName: "prx.org."
+      ValidationDomain: "prx.org"
+Resources:
+  Certificate:
+    Type: "AWS::CertificateManager::Certificate"
```

Comment:
I think we should do this as DNS validation, rather than email. [Ex](https://github.com/PRX/Infrastructure/blob/master/etc/wildcard-certificate.yml#L28)

Reply:
Ah, right.  Didn't know that needed a setting.